jetty-p2/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-project</artifactId>
+    <version>10.0.7-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>jetty-p2</artifactId>
+  <name>Jetty :: P2</name>
+  <description>Generates a (maven based) P2 Updatesite</description>
+  <packaging>pom</packaging>
+  <properties>
+    <tycho-version>2.5.0</tycho-version>
+  </properties>
+  <pluginRepositories>
+  <!-- TODO just until next release for testing latest changes -->
```

Comment:
@laeubi if tycho is finally released, do you still need this snapshot repo in here? I'd like to remove it if possible before release of jetty.

Reply:
@janbartel I did. then we can merge at the end of the build?